tests/test_login.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+
+from fxapom.fxapom import FxATestAccount
+from fxapom.fxapom import WebDriverFxA
+
+
+@pytest.mark.nondestructive
+class TestLogin(object):
+
+    _fxa_logged_in_indicator_locator = (By.ID, 'loggedin')
+
+    @pytest.mark.credentials
+    def test_existing_user_can_sign_in(self, mozwebqa):
```

Comment:
Another good point. I initially saw this as two different test cases, but what we are testing, in terms of this package, is identical. To have these two tests is really to be testing FxA itself, which is unnecessary.